.changeset/breezy-otters-rule.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": patch
+---
+
+Remove row gap from inline input grid when no secondary line is present.
```

Comment:
I think we should also mention at least a few components that are affected. E.g.:

```suggestion
Remove row gap from the inline `<InputGrid>` when no secondary line (`StatusMessage`) is present. Some other affected components: `LabeledInput`, `LabeledSelect`, `LabeledTextarea`.
```

Reply:
Updated both changesets.  Wasn't sure if I should be using React component names in the CSS changeset or not, did for now.

Reply:
The react changeset LGTM. For the css changeset, I think classnames would be better. So, something like:

```md
Remove unnecessary gap below the `.iui-input-grid` when `data-iui-label-placement='inline'` and no secondary line (`iui-status-message`) is present.
```

Reply:
@mayank99 said to not use class names within changesets: https://github.com/iTwin/iTwinUI/pull/2043#discussion_r1599072350

Reply:
Change made.